cleanlab/datalab/adapter/imagelab.py

```diff
@@ -0,0 +1,198 @@
+"""An internal wrapper around the Imagelab class from the CleanVision package to incorporate it into Datalab.
```

Comment:
What is the long-term plan to ensure the top-level datalab documentation is not full files that are only intended for internal use?   CC @elisno 

IMO eventually only datalab.py should be visible at the top-level datalab docs (since this is the only file intended for users to interface with). So everything else could go into a datalab/internal/ folder or datalab/utils/ or something like that so it is buried more deeply in the documentation rather than being visible at the top-level?

<img width="333" alt="Screen Shot 2023-07-16 at 11 14 21 AM" src="https://github.com/cleanlab/cleanlab/assets/1390638/bf0d8940-7096-4457-912e-3d8bedd3719f">